client/src/main/scala/grasshopper/client/parser/model/ParsedAddress.scala

```diff
@@ -0,0 +1,11 @@
+package grasshopper.client.parser.model
+
+case class AddressPart(
```

Comment:
Here's the full set of all possible address parts that may come through the the address parser:
* http://usaddress.readthedocs.org/en/latest/#details

You may want to just add all of them for now so the parse doesn't drop parts (or blow up?) when unexpected parts are present.

Another option would be to use the `validate=true` param, and then you'd get more of a guarantee that if all the parts are there it `OK`, otherwise `BadRequest`.  We might need to tune that functionality up a bit if we want to go that route.